### PR TITLE
PP-7687 Frontend production pipeline

### PR DESCRIPTION
**Pull request by mrlumbu**
Same as staging except without push to a further ECR at the end (this is the end of the line)

### CODE REVIEW

File: ci/pipelines/deploy-to-production.yml [hunk 1]
```diff
@@ -43,13 +51,15 @@ resources:
...
     source:
       repository: govukpay/toolbox
-      aws_access_key_id: ((readonly_access_key_id))
-      aws_secret_access_key: ((readonly_secret_access_key))
-      aws_session_token: ((readonly_session_token))
-      aws_role_arn: arn:aws:iam::((pay_aws_prod_account_id)):role/concourse
-      aws_ecr_registry_id: "((pay_aws_prod_account_id))"
-      aws_region: eu-west-1
       variant: release
+      <<: *aws_staging_config
```
Comment (oswaldquek) [inline ci/pipelines/deploy-to-production.yml:55]: this should be `aws_production_config`?

File: ci/pipelines/deploy-to-production.yml [hunk 2]
```diff
@@ -43,13 +51,15 @@ resources:
...
+  - name: frontend-ecr-registry-prod
+    type: registry-image-resource-1-1-0
+    icon: docker
+    source:
+      repository: govukpay/frontend
+      variant: release
+      <<: *aws_staging_config
```
Comment (oswaldquek) [inline ci/pipelines/deploy-to-production.yml:62]: same as above

File: ci/pipelines/deploy-to-production.yml [hunk 3]
```diff
@@ -15,6 +15,14 @@ definitions:
     ACCOUNT: production
     ENVIRONMENT: production-2
 
+  aws_staging_config: &aws_staging_config
+    aws_access_key_id: ((readonly_access_key_id))
+    aws_secret_access_key: ((readonly_secret_access_key))
+    aws_session_token: ((readonly_session_token))
+    aws_role_arn: arn:aws:iam::((pay_aws_staging_account_id)):role/concourse
+    aws_ecr_registry_id: "((pay_aws_staging_account_id))"
+    aws_region: eu-west-1
```
Comment (oswaldquek) [inline ci/pipelines/deploy-to-production.yml:24]: all these should be productionized

**Review by oswaldquek — CHANGES_REQUESTED** [review]
Would also be good to see a green build at https://cd.gds-reliability.engineering/teams/pay-deploy/pipelines/deploy-to-production?group=frontend before approving.

**Comment by mrlumbu** [comment]
sorry, bit of an end of day PR. Fixed

**Comment by mrlumbu** [comment]
The build will go green once your pay-infra changes are in master @oswaldquek

**Review by oswaldquek — APPROVED** [review]

**Pull request merged**